websites/A/AnimeKai/metadata.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "357878026429792258",
+    "name": ".waterboii"
+  },
+  "service": "AnimeKai",
+  "description": {
+    "en": "AnimeKai.to - A site to watch anime online for Free"
+  },
+  "url": "animekai.to",
+  "version": "1.0.1",
```

New activities should be 1.0.0
```suggestion
  "version": "1.0.0",
```